ferry.go

```diff
@@ -369,10 +349,10 @@ func checkConnection(logger *logrus.Entry, dsn string, db *sql.DB) error {
 	hasSSL := cipher != ""
 
 	logger.WithFields(logrus.Fields{
-		"hasSSL":     hasSSL,
-		"ssl_cipher": cipher,
-		"dsn":        dsn,
-	}).Infof("connected to %s", dsn)
+		"hasSSL":      hasSSL,
+		"ssl_cipher":  cipher,
+		"destination": destination,
```

Comment:
this dsn was pretty useful to have 🤔

Reply:
It still shows up, just in the line immediately above, which will be tagged with destination=source/target.